builder-support/debian/authoritative/debian-buster/config/ixfrdist.yml

```diff
@@ -0,0 +1,102 @@
+# ixfrdist example configuration file.
```

Comment:
Would it make sense to `sed` the example config into this at build-time?

Reply:
i had a look when introducing the file in debian, and it was too awkward, cannot remember the details.

What I'd ideally want is to tell ixfrdist "please use your good defaults, add these settings, and write the config file text to stdout".

Reply:
> What I'd ideally want is to tell ixfrdist "please use your good defaults, add these settings, and write the config file text to stdout".

When we introduce a new config parser, we'll do that